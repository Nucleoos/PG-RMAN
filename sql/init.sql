\! rm -rf results/init_test
\! pg_rman init -B ${PWD}/results/init_test --quiet;echo $?
\! find results/init_test | xargs ls -Fd | sort
\! pg_rman init -B ${PWD}/results/init_test --quiet;echo $?
