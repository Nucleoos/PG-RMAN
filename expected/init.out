\! rm -rf results/init_test
\! pg_rman init -B ${PWD}/results/init_test --quiet;echo $?
0
\! find results/init_test | xargs ls -Fd | sort
results/init_test/
results/init_test/backup/
results/init_test/backup/pg_xlog/
results/init_test/backup/srvlog/
results/init_test/pg_rman.ini
results/init_test/timeline_history/
\! pg_rman init -B ${PWD}/results/init_test --quiet;echo $?
ERROR: backup catalog already exist. and it's not empty.
2
